### ISSUE TITLE
@types endpoint: field default is not context sensitive

### ISSUE THREAD

**Issue by phgross** [4teamwork/opengever.core#5645]
_Harvest: `Projekt - Umsetzung Neues User Interface GEVER 2019/2020` - `Entwicklung`_

Steps to reproduce:
 - Create a repositoryfolder and set classification to `classified`
 - Make an api GET call `@types/opengever.dossier.businesscasedossier` on the created repositoryfolder
 
Actual result:
The global default value is returned
``` json
  "properties": {
         "classification": {
           ...
            "default": "unprotected",
           ...          
}
```

Expected result:
The context sensitive value should be returned:
``` json
  "properties": {
         "classification": {
           ...
            "default": "classified",
           ...          
}
```

Analysis:
plone.restapi's `DefaultJsonSchemaProvider` gets the default value for a field on an not bounded field (see DefaultJsonSchemaProvider).

**Comment by lukasgraf** [comment]
Fix in plone/plone.restapi#750

**Comment by phgross** [comment]
Has been fixed via #5725
